Pin Trivy version in `trivy.yaml` to fix Trivy vulnerability scanner workflow failures

**Describe the bug**
The `Trivy vulnerability scanner` workflow is failing in `.github/workflows/trivy.yaml` because `trivy-action` is used without an explicit `version`, and the pinned action commit defaults to `v0.65.0`. Like this: https://github.com/kaito-project/kaito/actions/runs/22559527276/job/65343117486

The failure happens during Trivy setup (before vulnerability scanning), so the job exits with code `1` even when image build succeeds.

**Steps To Reproduce**
Trigger the Trivy vulnerability scanner workflow

**Expected behavior**
Trivy vulnerability scanner workflow passed if there is no problem

**Logs**
```
Run echo "installing Trivy binary"
installing Trivy binary
aquasecurity/trivy info checking GitHub for tag 'v0.65.0'
aquasecurity/trivy info found version: 0.65.0 for v0.65.0/Linux/64bit
Error: Process completed with exit code 1.
```

I can make a PR to fix it.

--- a/.github/workflows/trivy.yaml
+++ b/.github/workflows/trivy.yaml
@@ -28,7 +28,6 @@ jobs:
             build_flags: "--target dependencies"
 
     steps:
-
       - name: Checkout code
         uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
 
@@ -43,6 +42,7 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@b6643a29fecd7f34b3597bc6acb0a98b03d33ff8 # v0.33.1
         with:
+          version: "v0.69.2"
           image-ref: ${{ matrix.image }}
           format: "table"
           exit-code: "1"

